salt/modules/archive.py

```diff
@@ -560,7 +570,7 @@ def unzip(zip_file, dest, excludes=None,
                             source = zfile.read(target)
                             os.symlink(source, os.path.join(dest, target))
                             continue
-                    zfile.extract(target, dest)
+                    zfile.extract(target, dest, password)
```

Comment:
I tested this to see what would happen if `password=None` and it seems to extract without a problem. So we're good!